Week1/homework/style.css

```diff
@@ -1 +1,81 @@
-/* add your styling here */
\ No newline at end of file
+  body {
+    background-color:  #212b37;
+    padding: 0px 0px 10px 0;
+    margin: auto;
+  }
+  
+  h1 {
+    text-align: center;
+    font-weight: bold;
+    font-size: 3em;
+    color: white;
+    font-family: Georgia, serif;
+    
+  }
+  
+  h2 {
+    text-align: center;
+    font-family: Impact, Charcoal, sans-serif;
+    font-size: 1.7em;
+    font-weight: normal;
+  }
+  
+  p {
+    font-family: "Comic Sans MS", cursive, sans-serif;
+    margin-left:20px;
+    font-weight: bolder;
+    }
+  
+.item {
+    float: left;
+    width: 23%;
+    height: 20em;
+    display: block;
+    background-color: rgb(255, 255, 255);
+    border-radius: 8px;
+    }
+  li.item{
```

Comment:
All `.items` are `li.items`. So you can remove the duplicate code here.
Interesting question: what would the values for display, margin and padding be in the browser with the css that you wrote?